docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,24 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]
+positional arguments:
+  command               Subcommand to run
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -g, --grpc_addr       GRPC_ADDR P4Runtime gRPC server address, format : <server IP>:<port>
+
+Details about supported option (``-g`` GRPC_ADDR)::
+---------------------------------------------------
+This is an optional parameter and default value is 127.0.0.1:9559. If P4Runtime server is not running
+locally, user can specify the address using this option.
+
+Eg: If server is listening on ``5.5.5.5:9559``, command to set pipeline will be:
+ ``p4rt-ctl -g 5.5.5.5:9559 set-pipe sample.pb.bin p4Info.txt``
+
+Details about supported commands::
```

Comment:
Same comment as earlier in the review: end the partial line with a colon or make the line a heading. Don't combine the two.